### ISSUE TITLE
Testing contiki-ng implementation which includes TSCH and the 6TiSCH with F-interop platform

### ISSUE THREAD

**Issue by tamouna** [contiki-ng/contiki-ng#602]
I'm trying to execute the test cases 6TiSCH implemented on F-interop platform.  I choosed contiki-ng implementation for running these tests. I got stuck on the 1st test case "TD_6TiSCH_SYN_01" which aims to check that a 6TiSCH node keep synchronized through pinging it.

I cannot ping the 6TiSCH node from my DAGroot. I flashed the code "simple-node" on my openmote cc2538 to act as a 6TiSCH node.

Here is the Test description of the first test case I want to execute :  
![image](https://user-images.githubusercontent.com/22098051/42580070-f630a41e-8529-11e8-9c11-db5b0ab9d7ca.png)

All the test cases are detailed here : https://openwsn-builder.paris.inria.fr/job/td-6tisch%20builder/lastSuccessfulBuild/artifact/6tisch_plugtest_td_june_2018.pdf 

Can you please help me to execute them by using contiki-ng implementation?

**Comment by yatch** [comment]
Can you try `node` under `examples/6tisch/etsi-plugtest-2017`? That is what we used in the plugtest in **2017**. Actually, I'm trying to do almost the same things here with `openmote-cc2538` as you're doing.

**Comment by tamouna** [comment]
Do I need to modify the node.c file for executing the first test case?

**Comment by yatch** [comment]
I don't think so as long as you can control your board via `shell`.

**Comment by tamouna** [comment]
I need to configure my 6TiSCH node to communicate on a single frequency. I'm using examples/6tisch/etsi-plugtest-2017 example.
In the available commands in the shell I didn't find the command I need to type for chosing my channel number.

**Comment by yatch** [comment]
@tamouna you're right; to change a channel or the channel hopping sequence, you need to edit `project-conf.h` and re-build your firmware. You may find other relevant settings in `project-conf.h` such as PAN-ID.

**Comment by tamouna** [comment]
did you manage to run the first test case?

**Comment by yatch** [comment]
No; I had a local issue on using my boards and I haven't had time to look into it...  :-/

**Comment by tamouna** [comment]
I modified the ` project-conf.h ` file but my DAGroot and my 6TiSCH node don't synchronized to each other. In wireshark I see that each one send an 'Enhanced Beacon'.

Can you send me the ` project-conf.h ` file modified with the appropriate parameters?

**Comment by yatch** [comment]
> I see that each one send an 'Enhanced Beacon'.

If you can see EBs sent by both of the nodes , they are supposed to be synchronized. Otherwise, both of them are working as DAGroot.

Again, I've not run the tests yet...

**Comment by tamouna** [comment]
@yatch when will you start to execute the test cases ? I can run them at the same time as you so you can help me to execute them.
